src/env.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018-2018 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import * as theia from '@wiptheia/plugin';
+import { COMMAND_NAME_PREFIX } from './common/constants';
+
+export function initEnvCommands(disposables: theia.Disposable[]) {
+    const showAllQueryParametersCommand = {
+        id: 'demo-all-query-parameters-command',
```

Comment:
IMHO the ids should be based on the name of the plugin
so if someone is created a demo plugin we're sure it's not colliding

and it will not collide with backend plugin as well

Reply:
Ok, I'll create another constant and will use it as prefix ini command ids

Reply:
yes, thanks.

we may as well use function (in case we share stuff between frontend and backend plugins)